catchup/catchpointService.go

```diff
@@ -523,6 +547,15 @@ func (cs *CatchpointCatchupService) processStageSwitch() (err error) {
 	return nil
 }
 
+// stopOrAbort is called when any of the stage processing function sees that cs.ctx has been canceled. It can be
+// due to the end user attempting to abort the current catchpoint catchup operation or due to a node shutdown.
+func (cs *CatchpointCatchupService) stopOrAbort() error {
```

Comment:
I understand that  StopOrAbort is meaningful only if it is preceded by an Abort or Stop calls. 
If this is correct, then the name StopOrAbort is misleading.

Reply:
I'm open to alternative naming for the function.

Reply:
how about AbortAndRevertNode ?